src/Table/Table.test.tsx

```diff
@@ -82,7 +82,7 @@ describe("Component: Table", () => {
     });
 
     it("Should render and be able to sort rows ", async () => {
-        const event: jest.Mock = jest.fn((rows: Array<TableRow>, sortByColumn: TableHeader) => console.log("onAfterSorting called"));
+        const event: jest.Mock = jest.fn((rows: Array<TableRow>, sortByColumn: TableHeader) => true);
```

Comment:
i think this can be simplified with `jest.fn()`

Reply:
Done